### PR TITLE
Create fasta export

### PR DESCRIPTION
**Pull request by blakesweeney**
This adds the sequence export logic to the pipeline. This will build a sequences directory with all files that we provide on the ftp site. It can be run with a simple `python -m luigi --module tasks.export.fasta FastaExport`.

For some this export fails in docker. Specifically the database disconnects when doing active sequence export. I have no idea why. It does manage to complete on the cluster though. While this is not ideal it probably can be merged. 

The approach is to use psql to run some COPY commands to produce temp files and then turn those files into fasta files. I don't use the copy_expert in the python library because it was failing in docker with a strange message. At least now when it fails in docker the message is clearer. I could return back to trying that approach though.

This adds Postgres to the docker file as we use psql to export the files. This means that using docker after this rebuilds the entire image which can take a bit. Be warned.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM centos:6.6
 
+RUN \
+    yum install -y 'https://download.postgresql.org/pub/repos/yum/9.5/redhat/rhel-6-x86_64/pgdg-centos95-9.5-3.noarch.rpm'
+
```
Comment (AntonPetrov) [inline Dockerfile]: Why not add this to the yum install below?

Comment (blakesweeney) [reply Dockerfile]: Makes more sense stylistically to me to keep them all together.

Comment (AntonPetrov) [reply Dockerfile]: I am wondering why not have a single yum install command to avoid creating an extra image layer?

Comment (blakesweeney) [reply Dockerfile]: That's a good point and I'll fix that now.

**Comment by blakesweeney** [comment]
I think this is ready to merge and I'd like to use some of this in solving https://github.com/RNAcentral/rnacentral-webcode/issues/288.

**Comment by AntonPetrov** [comment]
Are there any testing steps to make sure that the number of entries in the final FASTA files matches the expectation? We did run into problems like this in the past so might be good to prevent them from occurring in the future.

**Comment by blakesweeney** [comment]
Good point, I'll update the pull request with tests to ensure the counts of sequences for the export matches what we want.

**Comment by blakesweeney** [comment]
I've added some tests that hopefully will ensure we have the correct number of exports. One surprise is that I don't think we are exporting everything we should to the text search. At least the counts in the text search and the export differ.

**Pull request merged**